multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/Messages.java

```diff
@@ -91,6 +91,8 @@ public final class Messages {
     public static final String BUILDPACKS_REQUIRED_FOR_CNB = "Buildpacks must be provided when lifecycle is set to 'cnb'.";
     public static final String DOCKER_INFO_REQUIRED = "Docker information must be provided when lifecycle is set to 'docker'.";
     public static final String BUILDPACKS_NOT_ALLOWED_WITH_DOCKER = "Buildpacks must not be provided when lifecycle is set to 'docker'.";
+    public static final String EXTENSION_DESCRIPTORS_ARE_INVALID_SOON_WONT_WORK = "Extension descriptor(s) are invalid. These extension descriptors are now processed but soon they won't work and will lead to fail of the deploy. Fix them soon. Use at your own risk";
```

Comment:
I think that the message could be a bit more descriptive, this is a rough suggestion but I'm thinking something of this sort:
`"Extension descriptor(s) could not be parsed as a valid YAML file. These descriptors may fail future deployments once stricter validation is enforced. Please review and correct them now to avoid future issues.";`
Also it could be useful to add hints to what could be causing the issue to give directions (file type, extension etc.).